server/mdm/apple/cert.go

```diff
@@ -0,0 +1,123 @@
+package apple_mdm
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/micromdm/scep/v2/depot"
+)
+
+const defaultFleetDMAPIURL = "https://fleetdm.com"
```

Comment:
I think we might want to come to this later (in a separate PR), this has the same problem as our usage statistics cron: there's no way to point to a local development server to test without changing this URL (which makes me think if we could use the same http client for both in this future refactor)

having said that, testing this locally even if we can point to the right URL it's going to be complicated anyways.

Reply:
nvm, I just saw you can set this via an env variable. It would still be nice to use the same client, but not as important.

marking this as solved, sorry for the noise!